code/mission/missionbriefcommon.cpp

```diff
@@ -1819,50 +1818,6 @@ void brief_set_new_stage(vec3d *pos, matrix *orient, int time, int stage_num)
 	Last_new_stage = stage_num;
 }
 
-// ------------------------------------------------------------------------------------
-// camera_pos_past_target()
-//
-//
-int camera_pos_past_target(vec3d *start, vec3d *current, vec3d *dest)
-{
-	vec3d num, den;
-	float ratio;
-
-	vm_vec_sub(&num, current, start);
-	vm_vec_sub(&den, start, dest);
-
-	ratio = vm_vec_mag_quick(&num) / vm_vec_mag_quick(&den);
-	if (ratio >= 1.0f)
-		return TRUE;
-	
-	return FALSE;
-}
-
-/**
- * Interpolate between matrices.
- * elapsed_time/total_time gives percentage of interpolation between cur and goal.
- */
-void interpolate_matrix(matrix *result, matrix *goal, matrix *start, float elapsed_time, float total_time)
```

Comment:
Keep this function.  Move it to vecmat.

Reply:
Are you sure you want to keep it?
The implementation doesn't strike me as remarkable and the interface has problems.

Also it would be pretty trivial to reimplement if necessary.

Reply:
I'm not super attached to it, but it strikes me as very useful for physics, AI, and rotation code.  A good library function, in other words.  How is the interface problematic?

Reply:
Having two floats as input parameters instead of one float which is checked to be in [0, 1] is error prone.  
When goal and start are equal, nothing happens instead of the result being set to either one.

This seems a bit strange for a function which is effectively C = f * (A+B).

It probably made sense for the special purpose it was originally written for.

Any way: You call if you want to keep it, I wouldn't add it to vecmat in this PR though.

Reply:
On second thought, if it's in missionbriefcommon.cpp, it probably really is irrelevant.  Go ahead and remove it.